.github/workflows/ci.yml

```diff
@@ -209,38 +211,38 @@ jobs:
 
           # Setup Account
           echo "\nInstalling aws_account..."
-          psql postgres://postgres:test@localhost:5432/iasql -c "
+          psql "postgres://postgres:test@localhost:5432/iasql?ssl=true&sslmode=require" -c "
```

Comment:
maybe you can just have a var for postgres_connection_string, and reuse in all the following calls?

Reply:
Sounds good